core/streamdata.go

```diff
@@ -49,6 +49,12 @@ func (s *StreamParameters) StreamID() string {
 	return string(s.ManifestID) + "/" + s.RtmpKey
 }
 
+type SegmentParameters struct {
+	// From clip segment from this timestamp
```

Comment:
I think this comment is not clear, could you try to make it more comprehensible or just remove?